WordPress/src/main/java/org/wordpress/android/ui/posts/PostListAction.kt

```diff
@@ -91,5 +102,21 @@ fun handlePostListAction(
         is PostListAction.DismissPendingNotification -> {
             NativeNotificationsUtils.dismissNotification(action.pushId, activity)
         }
+        is PostListAction.CopyUrl -> {
+            try {
+                activity.clipboardManager?.setPrimaryClip(
+                        ClipData.newPlainText("${action.post.id}", action.post.link)
+                ) ?: throw NullPointerException("ClipboardManager is not supported on this device")
+
+                action.showSnackbar.invoke(action.messageSuccess)
+            } catch (e: Exception) {
+                 /**
+                  * Ignore any exceptions here as certain devices have bugs and will fail.
+                  * See https://crrev.com/542cb9cfcc927295615809b0c99917b09a219d9f for more info.
+                  */
```

Comment:
Thank you for the changes @ovitrif 🙇 
On a first pass everything looks good to me apart from a minor indentation issue (not related with the latest commits) in the comment that causes a `ktlint` error:
`Unexpected indentation (17) (it should be 16)`

Reply:
Thank you @antonis! I will fix this 🙇🏻‍♂️

Reply:
Checked with `ktlint`, fixed the formatting and ran `ktlint` again, now it's successful!

Merged with the latest develop too